### ISSUE TITLE
Bug: Purge old receipts do not delete the transactions.

### ISSUE THREAD

**Issue by MrDiba** [brarcher/budget-watch#153]
When pushing the "purge old receipts" button, selecting a date and pressing on clean the transactions will not disappear.

**Comment by brarcher** [comment]
Oh, this is the expected behavior. The largest potential storage used by the app is the receipt images. The purge option deletes the pictures (of receipts) from the transactions older than the given date, but still keeps the transactions themselves. The transaction data is much smaller in the database than the receipt images on the file system.

**Comment by MrDiba** [comment]
Ah okay, that makes sense. _In the next part I was trying to explain the problem I was facing and why I tried to dump the database but I then realized that I selected part of this month and part of the next month so it was showing two months worth of budgets. But because I would like to share my "problem" I would like to include it below._

The reason I tried to remove all the receipts is because when I switch my budget view to the next month the budgets will double and I tried to fix it by dumping the database.
View of this month.
![budget-watch1](https://user-images.githubusercontent.com/37332424/38071385-448e88de-3321-11e8-8210-5af303ff6436.png)

Showing the next month. _(This is actually a part of this month and the next month.)_
![budget-watch2](https://user-images.githubusercontent.com/37332424/38071560-4795b196-3322-11e8-99c2-5987574b04e6.png)

_A feature request I would like to propose is to being able to change the start of the month. But I think it is better to open a new issue for this. 
Thank you for your support and the great app!_
